projects/js-packages/ai-client/src/components/ai-control/index.tsx

```diff
@@ -79,11 +79,26 @@ export function AIControl(
 	ref
 ) {
 	const promptUserInputRef = useRef( null );
+	const InputWrapperRef = useRef( null );
 	const loading = state === 'requesting' || state === 'suggesting';
 
 	// Pass the ref to forwardRef.
 	useImperativeHandle( ref, () => promptUserInputRef.current );
 
+	const focusOnWrapper = () => {
+		// Small delay to avoid focus crash
+		setTimeout( () => {
+			InputWrapperRef.current?.focus?.();
+		}, 100 );
+	};
+
+	const handleSend = useCallback( () => {
+		onSend?.( value );
+		// The input is disabled while the request is executing, so we need
+		// to focus on the wrapper for the stop keyboard shortcut to work
```

Comment:
If the user clicks the AI Assistant wrapper, it still cancels the request, but clicking anywhere else outside of it the cancel shortcut won't work. I think it makes sense because we can have more than one assistants or forms requesting at the same time.

Reply:
yes, but it should be handled at block level. If the user selects the block it won't work I guess.